wordpress-mobile/gutenberg-mobile
# Implement video block settings

Added the missing Inspector Controls for video, except for the setting of the poster image that needs more custom work for native. 

![Simulator Screen Shot - iPhone 11 Pro Max - 2019-11-21 at 10 27 20](https://user-images.githubusercontent.com/651601/69329746-929b0400-0c49-11ea-8df9-4fc957c14262.png)

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/18660

To test:
 - Open the demo app
 - Tap on a video block
 - Tap on the settings icon (cog)
 - Check that the settings change the generated HTML

Update release notes:
